td.vue/src/desktop/menu.js

```diff
@@ -26,10 +27,10 @@ import spa from '@/i18n/es.js';
 // hide RUS & UKR for now: import ukr from '@/i18n/uk.js';
 import zho from '@/i18n/zh.js';
 
-const messages = { ara, deu, ell, eng, id, fin, fra, hin, por, spa, zho };
-// hide RUS & UKR for now: const messages = { ara, deu, ell, eng, id, fin, fra, hin, por, rus, spa, ukr, zho };
-const languages = [ 'ara', 'deu', 'ell', 'eng', 'id', 'fin', 'fra', 'hin', 'por', 'spa', 'zho' ];
-// hide RUS & UKR for now: const languages = [ 'ara', 'deu', 'ell', 'eng', 'id', 'fin', 'fra', 'hin', 'por', 'rus', 'spa', 'ukr', 'zho' ];
+const messages = { ara, deu, ell, eng, id, ms, fin, fra, hin, por, spa, zho };
```

Comment:
ideally both 'id' and 'ms' would be imported in alphabetical order, so could these both be moved to after 'hi'
and same for line 32